frontend/fingertips-frontend/components/pages/search/results/index.tsx

```diff
@@ -1,16 +1,52 @@
 'use client';
 
-import { BackLink, H1, Paragraph } from 'govuk-react';
+import {
+  BackLink,
+  H1,
+  ListItem,
+  Paragraph,
+  SectionBreak,
+  UnorderedList,
+} from 'govuk-react';
+import { SearchResult } from '@/components/molecules/Search/result';
 
-export function SearchResults({ indicator }: Readonly<{ indicator: string }>) {
+import { IndicatorSearchResult } from '@/app/search/results/search-result-data';
+
+type SearchResultsProps = {
+  indicator: string;
+  searchResults: IndicatorSearchResult[];
+};
+
+export default function SearchResults({
```

Comment:
Can we remove the default here too please

Reply:
done